Deploy-CAM.ps1

```diff
@@ -3455,6 +3484,9 @@ $vnetConfig.CSsubnetName = $ConnectionServiceSubnetName
 $vnetConfig.GWsubnetName = $GatewaySubnetName
 $vnetConfig.RWsubnetName = $RemoteWorkstationSubnetName
 
+# Set the minimum vCPUs needed for a full deployment or an add-on connection
+$minCoresFullDeploy = 6
```

Comment:
Maybe define these at the top of the file instead of hidden in the middle of the file?

Reply:
Checked with Bill and added a global variables section to the top.